### PR TITLE
added: logout user and improved routes code in api.php

### PR DESCRIPTION
**Pull request by HectorZR**
# User's logout

Reason:
Users need to close their sessions

Before pull request:
Users could not logout

After pull request:
Users can logout

Code Coverage & Build Info
Laravel

### CODE REVIEW

File: app/Http/Controllers/AuthController.php [hunk 1]
```diff
@@ -176,4 +177,23 @@ public function reset_password(Request $request)
...
+    /*
+     * Logout the current user session 
+     *
+     * @return Illuminate\Http\JsonResponse
+     */
+    public function logout()
+    {
+        if (!Auth::check())
+            return response([
+                'message' => trans('responses.AuthController.logout.error')
```
Comment (angel-afonso) [inline app/Http/Controllers/AuthController.php:190]: Is this necessary? I mean, the api middleware check if the token is active

**Review by angel-afonso — CHANGES_REQUESTED** [review]
Check the review please

File: app/Http/Controllers/AuthController.php (hunk 1 above)
Comment (HectorZR) [reply app/Http/Controllers/AuthController.php]: Yeah, I ran tests again and everything is working, you were right

**Review by angel-afonso — APPROVED** [review]
Excellent, i think we have to modify the login and register to return the refresh token with the response

**Pull request merged**